vembrane/modules/table.py

```diff
@@ -39,13 +39,6 @@ def add_subcommmand(subparsers):
         metavar="CHAR",
         help="Define the field separator (default: \\t).",
     )
-    parser.add_argument(
-        "--all",
-        "-a",
-        default=False,
-        action="store_true",
-        help="Do not filter duplicate entries.",
-    )
```

Comment:
Was this removal intentional?

Reply:
Yes, I was confused that the short options clashed (`-a`), then had a look where `--all` was used. Turns out: nowhere.